reflex/components/forms/button.py

```diff
@@ -29,21 +29,41 @@ class Button(ChakraComponent):
     loading_text: Var[str]
 
     # "lg" | "md" | "sm" | "xs"
-    size: Var[str]
-
+    size: Var[Literal["sm", "md", "lg", "xs"]]
     # "ghost" | "outline" | "solid" | "link" | "unstyled"
-    variant: Var[str]
+    variant: Var[Literal["ghost", "outline", "solid", "link", "unstyled"]]
 
     # Built in color scheme for ease of use.
     # Options:
     # "whiteAlpha" | "blackAlpha" | "gray" | "red" | "orange" | "yellow" | "green" | "teal" | "blue" | "cyan"
     # | "purple" | "pink" | "linkedin" | "facebook" | "messenger" | "whatsapp" | "twitter" | "telegram"
-    color_scheme: Var[str]
+    color_scheme: Var[
+        Literal[
+            "gray",
+            "red",
+            "orange",
+            "yellow",
+            "green",
+            "teal",
+            "blue",
+            "cyan",
+            "purple",
+            "pink",
+            "whiteAlpha",
+            "blackAlpha",
+            "linkedin",
+            "facebook",
+            "messenger",
+            "whatsapp",
+            "twitter",
+            "telegram",
+        ]
```

Comment:
i think we should move this to reflex/components/libs/chakra.py and import it where it's used.

Reply:
Do you mean refactoring this so to unpack (`Literal[*color_scheme])? If so, then it won't work as its only supported in python 3.11. I had this implemented initially but to revert as a  result. Alternatively, I figured using a Mixin class would work,